frame/support/src/storage/bounded_vec.rs

```diff
@@ -357,7 +357,7 @@ where
 		// BoundedVec<T, S> encodes like Vec<T> which encodes like [T], which is a compact u32
 		// plus each item in the slice:
 		// https://substrate.dev/rustdocs/v3.0.0/src/parity_scale_codec/codec.rs.html#798-808
-		codec::Compact::<u32>::max_encoded_len()
+		codec::Compact(S::get()).encoded_size()
```

Comment:
```suggestion
		codec::Compact(Seld::bound()).encoded_size()
```

might be deemed more consistent with the line that follows.

Reply:
Used `S::get` because that produces a `u32`, not a `usize`. The encoded size should probably be the same for both `Compact<u32>` and `Compact<usize>` for the same value, but it seemed better not to depend on that assumption.